src/client/Client.js

```diff
@@ -262,6 +303,8 @@ class Client extends BaseClient {
 
     this.emit(Events.DEBUG, 'Preparing to connect to the gateway...');
 
+    await this.updateClientBuildNumber();
```

Comment:
@TheDevYellowy it should become an option (user decides whether or not to use it)

Reply:
I see, I forgot that. One moment please